x/cheqd/types/keys.go

```diff
@@ -27,5 +27,5 @@ const (
 	DidCountKey = "did-count:"
 	// DidNamespaceKey FIXME: Should be `did-namespace:`.
 	// Networks was started with `testnet` value so we need a migration now.
-	DidNamespaceKey = "testnet"
+	DidNamespaceKey = "testnettestnet"
```

Comment:
Is this correct?

Reply:
Yep, it compensates this fix: https://github.com/cheqd/cheqd-node/pull/255/files#diff-abdef39a78c0c284466e9bf9b1c8510681273c75201228b173a6a6a1a466d989L11

Reply:
Maybe I'm being dense but explain me one last time tomorrow whether we still need this, given that both testnet and mainnet have this now set via genesis to "testnet" / "mainnet". If I understand correctly, this is internally stored as "testnettestnet" and "mainnetmainnet" but not exposed to the user. Fine. But given that both testnet and mainnet will be running exactly the same version of the software in the future, how will this validation account for that?